distribution/ddf-common/src/main/resources/etc/system.properties

```diff
@@ -137,6 +137,16 @@ org.codice.allowBasicAuthOverHttp=false
 
 # Sets the default port number for the catalog-ftp feature FTP endpoint
 org.codice.ddf.catalog.ftp.port=8021
+#
+# Maximum Endpoint Contact Interval
+#
+# Max seconds between attempts to contact the endpoint of a source
+org.codice.ddf.platform.util.http.maxRetryInterval=300
```

Comment:
We can definitely set the max retry interval ot 480. Scott suggested 5 minutes and that is where 300 came from. I'm not sure what you mean that we are never going to get 300?  I ran the unit test `testIsExponentialBackoff` and the interval sequence looks like:
```
0 = {Long@1077} "10"
1 = {Long@1078} "20"
2 = {Long@1079} "40"
3 = {Long@1080} "80"
4 = {Long@1081} "160"
5 = {Long@1082} "300"
6 = {Long@1083} "300"
7 = {Long@1084} "300"
8 = {Long@1085} "300"
...
```